packages/cli/src/Mfa/MfaController.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable import/no-cycle */
+import express from 'express';
+import * as speakeasy from 'speakeasy';
+import { Db } from '..';
+import type { AuthenticatedRequest } from '../requests';
+import type { MFA } from './types';
+import { v4 as uuid } from 'uuid';
+
+export const mfaController = express.Router();
```

Comment:
We are switching over to a new style for writing controllers.
I've converted this controller on a branch. please feel free to cherry-pick [these changes](https://github.com/n8n-io/n8n/compare/ado-21-sync-current-mfa-functionality-with...mfa-updates?expand=1).

Reply:
Ahh, yes, noticed it. I was going to move it to the new one before asking for a review. Thanks for adding it at once.